outline.md

```diff
@@ -0,0 +1,17 @@
+## Motivation
+
+Introduce the document by describing the motivations for producing a consolidated set of software citation principles. This section is basically an overview of the different use cases/scenarios where it would be desirable to have a way to cite software.
+
+## Related work
+
+In this section we discuss related work in the community. This is basically an overview of the documentation phased we went through as a group and will be heavily based on the collection of summaries we have of existing practices.
+
+## Software citation principles
+
+This section states for the first time the core software citation principles/architecture together with justification for each. The justification will likely be framed in a way such that it addresses the requirements as stated in the initial discussion of use cases/scenarios we wish to support.
+
+## Implementation examples
+
+For a subset (all?) of the use cases we then demonstrate how different stakeholders could adhere to/become compliant with the software citation principles. 
+
+This section is designed to offer explicit guidance for community members who wish to implement the software citation principles in their domain.
```

Comment:
> Would it make sense to frame this section as 'Examples and Recommendations'? Otherwise, I think this outline covers everything!

Yeah, renaming this section is probably a good idea. I see the `Software citation principles` section as the core deliverable and then the `Implementation examples` as the suggested ways to go about actually implementing them.

Thanks for the feedback @acabunoc :zap: